src/Cardano/Address.hs

```diff
@@ -184,3 +230,11 @@ invariantNetworkTag limit (NetworkTag num)
       ++ show num
       ++ ", but expected to be less than "
       ++ show limit
+
+-- | Encode a 'ByteString' in base16
+hex :: ByteString -> ByteString
+hex = convertToBase Base16
+
+-- | Decode a 'ByteString' from base16
+fromHex :: ByteString -> Either String ByteString
+fromHex = convertFromBase Base16
```

Comment:
I'll strip this one out as it isn't quite consistent with other similar encoding helpers (that are working directly from `Address`). Possibly, have this in a separate module, alongside `base58` and other commonly used encoding.

Reply:
Note, I also needed something similar on my branch, so I ended up splitting the code in a separate module: https://github.com/input-output-hk/cardano-addresses/blob/master/src/Codec/Binary/Encoding.hs

Reply:
reused Code.Binary.Encoding stuff in this commit https://github.com/input-output-hk/cardano-addresses/pull/31/commits/579d4b5dfc9916484cdd4aad1ec84388a3aeec08